stories/dynamic-side-content/dynamic-side-content.visual.js

```diff
@@ -0,0 +1,19 @@
+import * as stories from './dynamic-side-content.stories.js';
```

Comment:
We just moved to automated visual testing via chromatic 🎉 , so you can remove this file
Docs: https://github.com/SAP/fundamental-styles/wiki/Visual-testing-with-Chromatic
PR: https://github.com/SAP/fundamental-styles/pull/1584

Reply:
I removed `.visual.js`. Thank you!